website/docs/reference/resource-configs/quote_columns.md

```diff
@@ -8,13 +8,8 @@ default_value: false
 An optional seed configuration, used to determine whether column names in the seed file should be quoted when the <Term id="table" /> is created.
 
 * When `True`, dbt will quote the column names defined in the seed file when building a table for the seed, preserving casing.
-* (Default) When `False`, dbt will not quote the column names defined in the seed file.
-
-<Changelog>
-
-* Future: The default value may change in a future release. If you're using seed files, it is recommended that you set this configuration explicitly to avoid breaking changes in the future.
-
-</Changelog>
```

Comment:
@jtcohen6 I'm assuming this change already took place, but I didn't research to confirm/deny.

Do you agree with removing this section, or should we retain it?

Reply:
Correct! https://github.com/dbt-labs/dbt-core/pull/3918

Let's 🔪 it

Reply:
For completeness, this is the PR where this took place (first included in v1.0):
https://github.com/dbt-labs/dbt-core/pull/3918